src/ManifoldSampling.jl

```diff
@@ -11,10 +11,11 @@ function sampleTangent end
 
 # Sampling MKD
 function sampleTangent(M::AbstractDecoratorManifold, x::ManifoldKernelDensity, p=mean(x))
-    # get legacy matrix of coordinates and selected labels
-    coords, lbls = sample(x.belief,1)
-    X = hat(x.manifold, p, coords)
-    return X
+  # get legacy matrix of coordinates and selected labels
+  #TODO make sure that when `sample` is replaced in MKD, coordinates is a vector
+  coords, lbls = sample(x.belief,1)
+  X = hat(x.manifold, p, coords[:])
```

Comment:
`coords[:]` is the only change needed to fix the broken master.

Reply:
The current sampling state is:
- sampling from beliefs (eg `sample` and `rand`) are in coordinates.
- sampleTangent is on the tangent at point `p` or the identity element for groups.

Reply:
See JuliaRobotics/Caesar.jl#895